simple.js

```diff
@@ -705,6 +707,44 @@ define(function(require, exports, module) {
         }
 
         /**
+         * Simplifies Preview's UI and adds browser C9 command
+         */
+        function simplifyPreview() {
+            // add command to open URL
+            commands.addCommand({
+                name: "browser",
+                exec: function(args) {
+                    // ensure URL is given
+                    if (!_.isArray(args) || args.length !== 2 || !_.isString(args[1]))
+                        return console.log("Usage: c9 exec browser URL");
+
+                    // open URL in built-in browser tab
+                    preview.openPreview(args[1], null, true);
+                }
+            }, plugin);
+
+            // TODO determine if possible to do for "c9 exec browser" command only
```

Comment:
Was hoping to find a way to simplify only when the browser is opened through the command (as with phpliteadmin). The preview plugin also supports other previewers (e.g., markdown). Would be nice not to apply this simplification to them too.